src/main.ts

```diff
@@ -267,4 +266,9 @@ function isExportFormat(value: string): value is ExportFormat {
   return ['none', 'csv', 'shell', 'json', 'escape'].includes(value)
 }
 
+function getErrorMessage(error: unknown): string {
+  if (error instanceof Error) return error.message
+  return String(error)
```

Comment:
or `error.toString()`

Reply:
Thanks for your comment, I agree but i would prefer not to differ from the incoming commits from the original repository.